addons/web/static/lib/py.js/lib/py.js

```diff
@@ -969,6 +969,15 @@ var py = {};
         __init__: function () {
             this._values = [];
         },
+        __len__: function () {
+            return this._values.length;
+        },
+        __nonzero__: function () {
+            var len = this.__len__();
+            if (len === 0) return py.False;
+            if (len === 1 && py.PY_not(this._values[0])) return py.False;
```

Comment:
Oops, the python newb that I am needs to RTFM rather soon. Thanks for the clarification